script.js

```diff
@@ -1,12 +1,321 @@
+  let container = document.getElementById("root");
+
+  const allShows = getAllShows();
+  sortShowsByName(allShows);
+
+  let select = document.createElement("select");
+
+  let defaultOption = document.createElement("option");
+  defaultOption.innerHTML = "All Shows";
+  defaultOption.value = 0;
+  select.appendChild(defaultOption);
+
+  let option;
+  for (let show of allShows) {
+    option = document.createElement("option");
+    option.innerHTML = `${show.name}`;
+    option.value = show.id;
+    select.appendChild(option);
+  }
+  let dropMenuContainer = document.createElement("section");
+  dropMenuContainer.appendChild(select);
+
+  let searchContainer = document.createElement("section");
+  // searchContainer.classList.add("search-container");
+  searchContainer.id = "search-container";
+  searchContainer.appendChild(dropMenuContainer);
+
+  container.appendChild(searchContainer);
+ 
+
+  select.addEventListener("change", (event) => {
+    if (event.currentTarget.value == 0) {
+      document.getElementById("select-episode-element").remove();
+      document.getElementById("episode-searchbar").remove();
+      makeSearchForShows();
+      const allShows = getAllShows();
+      makePageForShows(allShows);
+    } else {
+      showSearchBarExist = document.getElementById("show-searchbar");
+      if (showSearchBarExist) {
+        document.getElementById("show-searchbar").remove();
+        makeSearchForEpisodes();
+      }
+    }
+
+    document.getElementById("container").remove();
+    fetchUrl(event.currentTarget.value);
+  });
+}
+
+//Episodes Functions
+function makePageForEpisodes(allEpisodes) {
+  let container = document.getElementById("root");
+
+  let episodesContainer = document.createElement("section");
+  episodesContainer.id = "container";
+  episodesContainer.classList.add("container-fluid");
+
+  let rowElem = document.createElement("div");
+  rowElem.classList.add("row");
+
+  for (let episode of allEpisodes) {
+    let eachEpisodeContainer = document.createElement("section");
+    eachEpisodeContainer.classList.add(
+      "bg-dark",
+      "text-white",
+      "each-episode",
+      "m-1"
+    );
+
+    let title = document.createElement("h4");
+    title.innerText = `${episode.name}`;
+    title.classList.add("title");
+
+    let seasonEpisodeNumber = document.createElement("h5");
+    seasonEpisodeNumber.innerText = makeIdForEpisodes(episode);
+
+    let imgContainer = document.createElement("div");
+    let image = document.createElement("img");
+
+    if (episode.image != null) {
+      image.src = episode.image.medium;
+      imgContainer.appendChild(image);
+    } else {
+      image.src = "./img/default-Episode.jpeg";
+      imgContainer.appendChild(image);
+    }
+    let episodeSummary = document.createElement("div");
+
+    if (episode.summary != null) {
+      episodeSummary.innerHTML = `${episode.summary}`;
+    } else {
+      episodeSummary.innerHTML = `No description found`;
+    }
+
```

Comment:
Good idea to add a message when there is no summary in episode 👍 
Also maybe it would be better to use `innerText` here since there is just plain text and no html tags? (not sure if it's super important though)